frontend/packages/pipelines-plugin/src/components/import/pipeline/pipeline-template-utils.ts

```diff
@@ -56,6 +59,8 @@ export const createPipelineForImportFlow = async (
         return { ...param, default: getImageUrl(name, namespace) };
       case 'DOCKERFILE':
         return { ...param, default: dockerfilePath };
+      case 'MAJOR_VERSION':
+        return { ...param, default: 'latest' }; // This needs to be removed when backend sends default value
```

Comment:
Probably should remove this 🤔  It's in the `preview` channel, we can install 4.6 which is 1.1.z and that will work fine. They need to update 1.2.z to have this default.

Reply:
removed it.